### PR TITLE
Autopsy scanners fit in surgery trays

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

This enables surgery trays to hold autopsy scanners

<details>
<summary>
Screenshot
</summary>

image
(Scanner not included)

</details>

## Why It's Good For The Game

Since it is a tool used in a surgery's step, I feel it makes sense that the autopsy tray (and surgery trays as a whole) should be able to hold this item and it potentially means less switching between containers.

## Changelog

:cl:
qol: The autopsy tray (and surgery trays) can now hold the autopsy scanner
/:cl:

### CODE REVIEW

**Comment by DATA-xPUNGED** [comment]
This doesn't seem to add an icon state to it?

**Comment by YehnBeep** [comment]
> This doesn't seem to add an icon state to it?

correct

**Comment by CoiledLamb** [comment]
> This doesn't seem to add an icon state to it?

i would complain, but it's both difficult enough to add a new one, and there's already a precedent for things not showing up, with medigel and the surgical masks, that it's not really a problem.

it does negate the purpose of the item somewhat, because now it could get stolen from the tray and you wouldn't know till you clicked on it, because no visible overlay, but ultimately that will probably not be a big deal

File: code/game/objects/items/surgery_tray.dm [hunk 1]
```diff
@@ -20,6 +20,7 @@
 		/obj/item/stack/sticky_tape/surgical,
 		/obj/item/surgical_drapes,
 		/obj/item/surgicaldrill,
+		/obj/item/autopsy_scanner,
```
Comment (san7890) [inline code/game/objects/items/surgery_tray.dm:23]: alphabetize your lists. there's an extension for VSCode that will automatically do this for you.

File: code/datums/storage/subtypes/surgery_tray.dm [hunk 2]
```diff
@@ -19,4 +19,5 @@
 		/obj/item/stack/sticky_tape/surgical,
 		/obj/item/surgical_drapes,
 		/obj/item/surgicaldrill,
+		/obj/item/autopsy_scanner,
```
Comment (san7890) [inline code/datums/storage/subtypes/surgery_tray.dm:22]: alphabetize.

**Review by san7890 — CHANGES_REQUESTED** [review]

**Pull request merged**